website/content/en/events/2024-cloud-native-sustainability-week.md

```diff
@@ -41,6 +41,7 @@ After the successful [Sustainability Week 2023](https://tag-env-sustainability.c
 | 11 | Stuttgart, Germany | TBD| TBD | Danijel Soldo, Stefan Bergstein
 | 12 | Paris, France | Oct. 15 | TBD | Smaine Kahlouch [Cloud Native Paris](https://community.cncf.io/paris/)
 | 13 | Brighton, UK | Oct. 16 | [Link](https://www.meetup.com/gsf-brighton/events/303193268) | Adam Newman, [Green Software Brighton](https://greensoftwarebrighton.co.uk)
+| 14 | Lorient, FR | Sept. 25 | [Link](https://community.cncf.io/events/details/cncf-lorient-presents-cloud-native-lorient-5/) | Sébastien DAVOULT [cloud Native Lorient](https://community.cncf.io/lorient/)
```

Comment:
@JulieChenadec thanks for the PR! Can you please also update the same file for different languages? Here's an example from the earlier PR: https://github.com/cncf/tag-env-sustainability/pull/505/files

Reply:
Thanks for letting me know. I just did it! Except for DE website, because there is no 2024 edition